in-review/front-end-javascript/getting-started-with-angularjs/article.md

```diff
@@ -155,16 +150,18 @@ for Example :
 <button ng-click="functionalExpressions()">Click me !</button>
 <input ng-model="name" />
 ```
- Anything written in `input tag` will observe by scope api `$watch` and will display value of `$scope.name` object in paragraph tag.
+Anything written in `input tag` will be observed by scope API `$watch` and will display the value of the `$scope.name` object in paragraph tag.
 ```html
 <p>Hello {{name}}!</p>
 ```
 
 #### Source Code
-[plunker](https://plnkr.co/edit/5sxudDgiAOGe8hKYjRLK?p=preview) for live example.
+See [plunker](https://plnkr.co/edit/5sxudDgiAOGe8hKYjRLK?p=preview) for live example.
 
 #### ScreenShot
 ![description](https://raw.githubusercontent.com/pluralsight/guides/master/images/aeeb2d6b-58b1-4d24-97cc-9c42d0ef7954.png)
 
 
-This is how we implement angular.module, register ngController to angular module and use of scope variable with controller and view to binding with ngModel in  AngularJS. see you soon with next article.
\ No newline at end of file
+This is how we implement `angular.module`, register ngController to the Angular module, and use the scope variable with the controller and the view to bind with AngularJS's ngModel. 
+
+I hope you found this article infomrative! See you soon with my next article on AngularJS2. 
```

Comment:
Please change AngularJS2 with AngularJS, since the guide is based on AngularJS version 1.4.x
I tried to made changes but I do not have push permission for this repository.